app/views/Layout.scala.html

```diff
@@ -14,29 +14,32 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.{HmrcStandardFooter, HmrcStandardHeader, HmrcHead, HmrcScripts}
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukLayout
+@import play.api.i18n.Messages
+@import play.api.mvc.RequestHeader
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.backlink.BackLink
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers._
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.hmrcstandardpage.{HmrcStandardPageParams, TemplateOverrides}
 
 @this(
-    govukLayout: GovukLayout,
-    hmrcHead: HmrcHead,
-    hmrcStandardHeader: HmrcStandardHeader,
-    hmrcStandardFooter: HmrcStandardFooter,
-    hmrcScripts: HmrcScripts
+        hmrcLayout: HmrcStandardPage,
 )
 
 @(
-    pageTitle: String,
-    headBlock: Option[Html] = None,
-    beforeContentBlock: Option[Html] = None,
-    scripts: Option[Html] = None
+        pageTitle: String,
+        headBlock: Option[Html] = None,
+        beforeContentBlock: Option[Html] = None,
+        scripts: Option[Html] = None,
+        backLink: Option[BackLink] = None
 )(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
-@govukLayout(
-  pageTitle = Some(pageTitle),
-  headBlock = Some(hmrcHead(headBlock)),
-  headerBlock = Some(hmrcStandardHeader()),
-  beforeContentBlock = Some(hmrcHead(beforeContentBlock)),
-  scriptsBlock = Some(hmrcScripts(scripts)),
-  footerBlock = Some(hmrcStandardFooter())
+@hmrcLayout(
+    HmrcStandardPageParams(
```

Comment:
You might need to define the service Url and sign out url.

Reply:
I'm gonna declare that one out of scope for this PR as its not been defined in the ticket or the wireframes - there are also other tickets which may cover this off